BUG: bilibili 动态新类型 `动态充电互动抽奖` 解析异常

日志：
```
05-27 14:20:11 [ERROR] src | SemaphoreGather | Task(s) called by "bili_dynamic_update_monitor" in "/opt/mcsm-apps/omega-miya/src/plugins/bilibili_dynamic_monitor/monitor.py" raised ValidationError exception in task(1): 39 validation errors for Dynamics
data.items.0.DynForwardItem.modules.module_dynamic.additional.ModuleDynamicAdditionalNone.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynForwardItem.modules.module_dynamic.additional.ModuleDynamicAdditionalNone.none
  Field required [type=missing, input_value={'type': 'ADDITIONAL_TYPE... 2, 'upower_level': 10}}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynForwardItem.modules.module_dynamic.additional.ModuleDynamicAdditionalPgc.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynForwardItem.modules.module_dynamic.additional.ModuleDynamicAdditionalPgc.pgc
  Field required [type=missing, input_value={'type': 'ADDITIONAL_TYPE... 2, 'upower_level': 10}}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynForwardItem.modules.module_dynamic.additional.ModuleDynamicAdditionalGoods.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynForwardItem.modules.module_dynamic.additional.ModuleDynamicAdditionalGoods.goods
  Field required [type=missing, input_value={'type': 'ADDITIONAL_TYPE... 2, 'upower_level': 10}}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynForwardItem.modules.module_dynamic.additional.ModuleDynamicAdditionalVote.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynForwardItem.modules.module_dynamic.additional.ModuleDynamicAdditionalVote.vote
  Field required [type=missing, input_value={'type': 'ADDITIONAL_TYPE... 2, 'upower_level': 10}}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynForwardItem.modules.module_dynamic.additional.ModuleDynamicAdditionalCommon.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynForwardItem.modules.module_dynamic.additional.ModuleDynamicAdditionalCommon.common
  Field required [type=missing, input_value={'type': 'ADDITIONAL_TYPE... 2, 'upower_level': 10}}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynForwardItem.modules.module_dynamic.additional.ModuleDynamicAdditionalMatch.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynForwardItem.modules.module_dynamic.additional.ModuleDynamicAdditionalMatch.match
  Field required [type=missing, input_value={'type': 'ADDITIONAL_TYPE... 2, 'upower_level': 10}}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynForwardItem.modules.module_dynamic.additional.ModuleDynamicAdditionalUpRcmd.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynForwardItem.modules.module_dynamic.additional.ModuleDynamicAdditionalUpRcmd.up_rcmd
  Field required [type=missing, input_value={'type': 'ADDITIONAL_TYPE... 2, 'upower_level': 10}}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynForwardItem.modules.module_dynamic.additional.ModuleDynamicAdditionalReserve.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynForwardItem.modules.module_dynamic.additional.ModuleDynamicAdditionalReserve.reserve
  Field required [type=missing, input_value={'type': 'ADDITIONAL_TYPE... 2, 'upower_level': 10}}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynForwardItem.modules.module_dynamic.additional.ModuleDynamicAdditionalUgc.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynForwardItem.modules.module_dynamic.additional.ModuleDynamicAdditionalUgc.ugc
  Field required [type=missing, input_value={'type': 'ADDITIONAL_TYPE... 2, 'upower_level': 10}}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynForwardItem.modules.module_dynamic.additional.BaseModuleDynamicAdditional.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynForwardItem.orig
  Field required [type=missing, input_value={'basic': {'comment_id_st..._DRAW', 'visible': True}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynCommonItem.modules.module_dynamic.additional.ModuleDynamicAdditionalNone.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynCommonItem.modules.module_dynamic.additional.ModuleDynamicAdditionalNone.none
  Field required [type=missing, input_value={'type': 'ADDITIONAL_TYPE... 2, 'upower_level': 10}}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynCommonItem.modules.module_dynamic.additional.ModuleDynamicAdditionalPgc.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynCommonItem.modules.module_dynamic.additional.ModuleDynamicAdditionalPgc.pgc
  Field required [type=missing, input_value={'type': 'ADDITIONAL_TYPE... 2, 'upower_level': 10}}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynCommonItem.modules.module_dynamic.additional.ModuleDynamicAdditionalGoods.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynCommonItem.modules.module_dynamic.additional.ModuleDynamicAdditionalGoods.goods
  Field required [type=missing, input_value={'type': 'ADDITIONAL_TYPE... 2, 'upower_level': 10}}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynCommonItem.modules.module_dynamic.additional.ModuleDynamicAdditionalVote.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynCommonItem.modules.module_dynamic.additional.ModuleDynamicAdditionalVote.vote
  Field required [type=missing, input_value={'type': 'ADDITIONAL_TYPE... 2, 'upower_level': 10}}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynCommonItem.modules.module_dynamic.additional.ModuleDynamicAdditionalCommon.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynCommonItem.modules.module_dynamic.additional.ModuleDynamicAdditionalCommon.common
  Field required [type=missing, input_value={'type': 'ADDITIONAL_TYPE... 2, 'upower_level': 10}}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynCommonItem.modules.module_dynamic.additional.ModuleDynamicAdditionalMatch.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynCommonItem.modules.module_dynamic.additional.ModuleDynamicAdditionalMatch.match
  Field required [type=missing, input_value={'type': 'ADDITIONAL_TYPE... 2, 'upower_level': 10}}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynCommonItem.modules.module_dynamic.additional.ModuleDynamicAdditionalUpRcmd.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynCommonItem.modules.module_dynamic.additional.ModuleDynamicAdditionalUpRcmd.up_rcmd
  Field required [type=missing, input_value={'type': 'ADDITIONAL_TYPE... 2, 'upower_level': 10}}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynCommonItem.modules.module_dynamic.additional.ModuleDynamicAdditionalReserve.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynCommonItem.modules.module_dynamic.additional.ModuleDynamicAdditionalReserve.reserve
  Field required [type=missing, input_value={'type': 'ADDITIONAL_TYPE... 2, 'upower_level': 10}}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynCommonItem.modules.module_dynamic.additional.ModuleDynamicAdditionalUgc.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.items.0.DynCommonItem.modules.module_dynamic.additional.ModuleDynamicAdditionalUgc.ugc
  Field required [type=missing, input_value={'type': 'ADDITIONAL_TYPE... 2, 'upower_level': 10}}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.10/v/missing
data.items.0.DynCommonItem.modules.module_dynamic.additional.BaseModuleDynamicAdditional.type
  Input should be 'ADDITIONAL_TYPE_NONE', 'ADDITIONAL_TYPE_PGC', 'ADDITIONAL_TYPE_GOODS', 'ADDITIONAL_TYPE_VOTE', 'ADDITIONAL_TYPE_COMMON', 'ADDITIONAL_TYPE_MATCH', 'ADDITIONAL_TYPE_UP_RCMD', 'ADDITIONAL_TYPE_UGC' or 'ADDITIONAL_TYPE_RESERVE' [type=enum, input_value='ADDITIONAL_TYPE_UPOWER_LOTTERY', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
```

--- a/src/utils/bilibili_api/models/dynamic.py
+++ b/src/utils/bilibili_api/models/dynamic.py
@@ -91,16 +91,16 @@ class EmojiType(StrEnum):
 @unique
 class AdditionalType(StrEnum):
     """相关内容卡片类型"""
-    none = 'ADDITIONAL_TYPE_NONE' # 无附加类型
-    pgc = 'ADDITIONAL_TYPE_PGC' # 番剧影视
+    none = 'ADDITIONAL_TYPE_NONE'  # 无附加类型
+    pgc = 'ADDITIONAL_TYPE_PGC'  # 番剧影视
     goods = 'ADDITIONAL_TYPE_GOODS'  # 商品信息
     vote = 'ADDITIONAL_TYPE_VOTE'  # 投票
     common = 'ADDITIONAL_TYPE_COMMON'  # 一般类型
-    match = 'ADDITIONAL_TYPE_MATCH' # 比赛
-    up_rcmd = 'ADDITIONAL_TYPE_UP_RCMD' # UP主推荐
+    match = 'ADDITIONAL_TYPE_MATCH'  # 比赛
+    up_rcmd = 'ADDITIONAL_TYPE_UP_RCMD'  # UP主推荐
     ugc = 'ADDITIONAL_TYPE_UGC'  # 视频跳转
-    reserve = 'ADDITIONAL_TYPE_RESERVE' # 直播预约
-    upower_lottery = 'ADDITIONAL_TYPE_UPOWER_LOTTERY' # 动态充电互动抽奖
+    reserve = 'ADDITIONAL_TYPE_RESERVE'  # 直播预约
+    upower_lottery = 'ADDITIONAL_TYPE_UPOWER_LOTTERY'  # 动态充电互动抽奖
 
 
 @unique
@@ -297,8 +297,10 @@ class DynItemModuleAuthor(BaseBilibiliModel):
 
 
 class BaseAdditionalItemDesc(BaseBilibiliModel):
-    style: int
-    text: str
+    """相关内容卡片描述信息字段"""
+    style: int = Field(default=-1)
+    text: str = Field(default_factory=str)
+    jump_url: str = Field(default_factory=str)
 
 
 class AdditionalNoneItem(BaseBilibiliModel):
@@ -378,8 +380,18 @@ class AdditionalReserveItem(BaseBilibiliModel):
     title: str
     up_mid: str
 
+
 class AdditionalUpowerLotteryItem(BaseBilibiliModel):
     """动态充电互动抽奖"""
+    desc: BaseAdditionalItemDesc
+    hint: BaseAdditionalItemDesc
+    jump_url: str
+    rid: str
+    state: int
+    title: str
+    up_mid: str
+    upower_action_state: int
+    upower_level: int
 
 
 class BaseModuleDynamicAdditional(BaseBilibiliModel):
@@ -430,6 +442,7 @@ class ModuleDynamicAdditionalUgc(BaseModuleDynamicAdditional):
 class ModuleDynamicAdditionalReserve(BaseModuleDynamicAdditional):
     """预约信息"""
     reserve: AdditionalReserveItem
+
 
 class ModuleDynamicUpowerLottery(BaseModuleDynamicAdditional):
     """动态充电互动抽奖"""
@@ -671,7 +684,6 @@ class ModuleDynamicMajorNone(BaseModuleDynamicMajor):
 
     def get_major_text(self) -> str:
         return self.none.tips
-
 
 
 class ModuleDynamicMajorOpus(BaseModuleDynamicMajor):

--- a/src/utils/bilibili_api/models/dynamic.py
+++ b/src/utils/bilibili_api/models/dynamic.py
@@ -91,15 +91,16 @@ class EmojiType(StrEnum):
 @unique
 class AdditionalType(StrEnum):
     """相关内容卡片类型"""
-    none = 'ADDITIONAL_TYPE_NONE'
-    pgc = 'ADDITIONAL_TYPE_PGC'
+    none = 'ADDITIONAL_TYPE_NONE' # 无附加类型
+    pgc = 'ADDITIONAL_TYPE_PGC' # 番剧影视
     goods = 'ADDITIONAL_TYPE_GOODS'  # 商品信息
     vote = 'ADDITIONAL_TYPE_VOTE'  # 投票
     common = 'ADDITIONAL_TYPE_COMMON'  # 一般类型
-    match = 'ADDITIONAL_TYPE_MATCH'
-    up_rcmd = 'ADDITIONAL_TYPE_UP_RCMD'
+    match = 'ADDITIONAL_TYPE_MATCH' # 比赛
+    up_rcmd = 'ADDITIONAL_TYPE_UP_RCMD' # UP主推荐
     ugc = 'ADDITIONAL_TYPE_UGC'  # 视频跳转
-    reserve = 'ADDITIONAL_TYPE_RESERVE'
+    reserve = 'ADDITIONAL_TYPE_RESERVE' # 直播预约
+    upower_lottery = 'ADDITIONAL_TYPE_UPOWER_LOTTERY' # 动态充电互动抽奖
 
 
 @unique
@@ -377,6 +378,8 @@ class AdditionalReserveItem(BaseBilibiliModel):
     title: str
     up_mid: str
 
+class AdditionalUpowerLotteryItem(BaseBilibiliModel):
+    """动态充电互动抽奖"""
 
 
 class BaseModuleDynamicAdditional(BaseBilibiliModel):
@@ -428,6 +431,10 @@ class ModuleDynamicAdditionalReserve(BaseModuleDynamicAdditional):
     """预约信息"""
     reserve: AdditionalReserveItem
 
+class ModuleDynamicUpowerLottery(BaseModuleDynamicAdditional):
+    """动态充电互动抽奖"""
+    upower_lottery: AdditionalUpowerLotteryItem
+
 
 type ModuleDynamicAdditional = (
         ModuleDynamicAdditionalNone
@@ -439,6 +446,7 @@ type ModuleDynamicAdditional = (
         | ModuleDynamicAdditionalUpRcmd
         | ModuleDynamicAdditionalReserve
         | ModuleDynamicAdditionalUgc
+        | ModuleDynamicUpowerLottery
         | BaseModuleDynamicAdditional
 )
 

